aikau/src/main/resources/alfresco/forms/controls/BaseFormControl.js

```diff
@@ -1261,14 +1278,23 @@ define(["dojo/_base/declare",
                this.deferredValuePublication.resolve();
             }
 
-            on(this.domNode, "click", lang.hitch(this, this.suppressFocusRequest));
+            if (this.usedInList)
+            {
+               on(this.domNode, "click", lang.hitch(this, this.suppressFocusRequest));
+            }
          }
          else
          {
             // No action yet. Don't unsubscribe.
          }
       },
 
+      suppressFocusRequest: function alfresco_lists_KeyboardNavigationSuppressionMixin__suppressFocusRequest(evt) {
+         evt.stopPropagation();
+         evt.preventDefault();
```

Comment:
We should try to ensure we only use one of these, generally `stopPropagation()`, otherwise we're adding redundant code. Just best-practice rather than anything wrong with what's present.